pkg/sdk/testint/schemas_integration_test.go

```diff
@@ -156,25 +281,100 @@ func TestInt_SchemasAlter(t *testing.T) {
 		assert.Equal(t, table.Name, schemaDetails[0].Name)
 	})
 
-	t.Run("set", func(t *testing.T) {
```

Comment:
do I see correctly that the previous test for set was replaced with the parameters one? The only one non-parameter attribute is comment, right? Even if this is the only one, let's test set + unset for it

Reply:
Added.